src/statsmodels/statsmodel.jl

```diff
@@ -86,6 +86,16 @@ function StatsBase.predict(mm::DataFrameRegressionModel, df::AbstractDataFrame;
     return(out)
 end
 
+# Predict function with confidence intervars
+function StatsBase.predict(mm::DataFrameRegressionModel, df::AbstractDataFrame, interval_type::Symbol, level::Real = 0.95; kwargs...)
```

Comment:
I am not sure - because the method with interval_type returns a Matrix, the version without returns a Vector. Having separate methods is necessary for type stability, is it not?